### PR TITLE
Site is migrated to Gatsby from reactJS

### LINKED ISSUE #130: Site is not working as there is a bug in migrating it from React to gatsby
#### Current Behavior

When we run the command `npm start` in _/site_ directory, it throws an error even though we install all the dependencies

#### Expected Behavior

Entire site should be migrated to gatsby and it should work fine

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by gopivaibhav**
**Description**

This PR fixes #130

**Notes for Reviewers**
Screenshot from 2023-08-08 00-04-06

Screenshot from 2023-08-08 00-04-48

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: site/src/pages/404.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import { graphql } from "gatsby"
+
+
+const NotFoundPage = ({ data, location }) => {
+  const siteTitle = data.site.siteMetadata.title
```
Comment (leecalcote) [inline site/src/pages/404.js:6]: Unused variable...

**Review by leecalcote — APPROVED** [review]
A few lint issues to address, but looking good overall, @gopivaibhav 👍

**Comment by leecalcote** [comment]
Good work!

File: site/package.json [hunk 2]
```diff
@@ -4,6 +4,17 @@
   "private": true,
   "description": "layer5-recognition-program",
   "author": "Lee Calcote",
+  "dependencies": {
+    "@parcel/watcher-darwin-x64": "^2.3.0-alpha.1",
```
Comment (Nikhil-Ladha) [inline site/package.json:8]: Why was this package added?

File: site/src/pages/404.js (hunk 1 above)
Comment (gopivaibhav) [reply site/src/pages/404.js:6]: Oh yes. Actually, I just used a starter template of Gatsby to migrate. So, I overlooked at it.

**Comment by aabidsofi19** [comment]
Good work , I will look at the  changes with fresh eyes in morning

**Pull request merged**